WordPress/Classes/ViewRelated/System/RotationAwareNavigationViewController.swift

```diff
@@ -0,0 +1,25 @@
+import UIKit
+import WordPressShared
+
+
+class RotationAwareNavigationViewController: UINavigationController {
+
+    override var shouldAutorotate: Bool {
+        // Respect the top child's orientation prefs.
+        guard let shouldAutorotate = topViewController?.shouldAutorotate else {
```

Comment:
We could just do `return topViewController?.shouldAutoRotate ?? super.shouldAutorotate` here?

Reply:
AHHHHH how could i miss that!!! thanks for the comment, beautiful!!